math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -21,6 +21,8 @@
    #define _VECOPS_USE_EXTERN_TEMPLATES true
 #endif
 
+#define _USE_MATH_DEFINES // enable definition of M_PI
```

Comment:
I presume that define is there because any of the next headers can bring cmath for non-modules. We should probably immediately after the include of cmath undefine it.